strawberryfields/engine.py

```diff
@@ -547,8 +547,8 @@ def run(self, program: Program, *, compile_options=None, **kwargs) -> Optional[R
 
                 if job.status == "failed":
                     message = (
-                        "The remote job %s failed due to an internal "
-                        "server error. Please try again." % job.id
+                        "The remote job {} failed due to an internal "
+                        "server error. Please try again. {}".format(job.id, job.meta)
```

Comment:
These were previously made so such that they can be used for logging, but here the message is already pre-defined, so I guess `format` should work just fine as well?